release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
++
+image::ose32-relnote-resource-limits.png["Set Resource Limits"]
+- A form-based editor for build configurations has been added for modifying
+commonly edited fields directly from the web console.
++
+image::ose32-relnote-edit-bc.png["Edit Build Configurations"]
+- All *Browse* resource pages (e.g, viewing a particular pod) now have a tab for
+*Events* related to that pod.
+- Limits, quotas, and quota scopes are now displayed.
+- More error and warning information is now displayed about routes, their
+configuration, and their use in the system.
+- Support has been added for filtering and sorting on all *Events* pages.
+- You can now edit a project's display name and description from the *Settings*
+page.
+- Existing persistent volume claims (PVCs) can now be listed and attached to
+deployments and deployment configurations.
+- More detailed pod status is now provided on all pages.
+- Better status and alert messages are now provided.
+- Improved *_Dockerfile_* build keyword highlighting has been added when editing
+builds.
+- More accurate information is now displayed about routes based on which addresses
+the router exposed them under.
+- The layout and display of logs have been improved.
+
+==== Developer CLI
+
+- The following commands have been added to `oc create`, allowing more objects to
+be created directly using the CLI (instead of passing it a file or JSON/YAML):
++
+[options="header",cols="1,3"]
+|====
+|Command          |Description
+
+|`namespace`
+|Create a namespace with the specified name.
+
+|`secret`
+|Create a secret using a specific subcommand (`docker-registry` or `generic`).
+
+|`configmap`
+|Create a `*ConfigMap*` from a local file, directory, or literal value.
+
+|`serviceaccount`
+|Create a service account with the specified name.
+
+|`route`
+|Expose containers externally via secured routes. Use the `edge`, `passthrough`,
+or `reencrypt` subcommands and specify the secret values to be used for the
+route.
+|====
+- Display more information about the application being created in `oc new-app`,
+including any display name or description set on the image as a label, or
+whether the image may require running as root.
+- If you have set up the *latest* tag in an image stream to point to another tag
+in the same image stream, the `oc new-app` command will follow that reference
+and create the application using the referenced tag, not *latest*. This allows
+administrators to ensure applications are created on stable tags (like
+*php:5.6*). The default image streams created in the *openshift* project follow
+this pattern.
+- You can view the logs of the oldest pod in a deployment or build configuration
+with `oc logs dc/<name>`.
+- The `oc env` and `oc volume` commands have been moved to `oc set env` and `oc
+set volume`, and future commands that modify aspects of existing resources will
+be located under this command.
+- When a pod is crashlooping, an error is now displayed in `oc status` and
```

Comment:
Updated:

> When a pod is crash-looping, meaning it is starting and exiting repeatedly, an
error is now displayed in `oc status` and provides more information about
possible causes.